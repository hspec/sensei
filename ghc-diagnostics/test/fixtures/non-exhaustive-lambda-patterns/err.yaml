version: '1.1'
ghcVersion: ghc-9.10.0
span:
  file: test/fixtures/non-exhaustive-lambda-patterns/Foo.hs
  start:
    line: 5
    column: 7
  end:
    line: 5
    column: 13
severity: Error
code: 62161
message:
- |-
  Pattern match(es) are non-exhaustive
  In a \case alternative:
      Patterns of type `Foo' not matched:
          Foo
          Bar _
          Baz _ _
hints: []
reason:
  flags:
  - incomplete-patterns
