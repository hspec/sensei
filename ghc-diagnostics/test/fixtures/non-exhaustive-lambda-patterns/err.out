test/fixtures/non-exhaustive-lambda-patterns/Foo.hs:5:7: error: [GHC-62161] [-Wincomplete-patterns, Werror=incomplete-patterns]
    Pattern match(es) are non-exhaustive
    In a \case alternative:
        Patterns of type `Foo' not matched:
            Foo
            Bar _
            Baz _ _

